db/db_impl/db_impl_debug.cc

```diff
@@ -122,6 +122,16 @@ Status DBImpl::TEST_FlushMemTable(bool wait, bool allow_write_stall,
   return FlushMemTable(cfd, fo, FlushReason::kTest);
 }
 
+Status DBImpl::TEST_FlushMemTable(ColumnFamilyData* cfd,
+                                  const FlushOptions& flush_opts) {
+  return FlushMemTable(cfd, flush_opts, FlushReason::kTest);
```

Comment:
Isn't this usage the same as the other (already exisitng) `TEST_FlushMemTable`? Curious why need a new override. 

In the newly created `CFDropRaceWithWaitForFlushMemTables` test, would calling like this work? 
```
ASSERT_NOK(dbfull()->TEST_FlushMemTable(false, true, handles_[1]));
```

Reply:
We cannot use `TEST_FlushMemTable` because it uses `ColumnFamilyHandle` which will be deleted in our tests. We have to use ColumnFamilyData.

Reply:
To add to previous comment, this line will result in a segfault should we use `TEST_FlushMemTable(ColumnFamilyHandle* )`. https://github.com/facebook/rocksdb/blob/master/db/db_impl/db_impl_debug.cc#L120

Reply:
Thanks, that explains it. I was sure I was missing something in this specific function; should have looked more holistically.

Reply:
Thanks for the review! Maybe I can add a comment to the .h file explaining why we need it.